tools/ci/run-examples.sh

```diff
@@ -63,7 +65,19 @@ done
 echo "Stopping standalone Flink cluster."
 ./flink-"${FLINK_VERSION}"/bin/stop-cluster.sh
 
-python -m pip install "${wheel_file}[spark]"
+echo "Downloading Spark."
+curl -LO https://archive.apache.org/dist/spark/spark-"${SPARK_VERSION}"/spark-"${SPARK_VERSION}"-bin-hadoop3.tgz
+tar -xzf spark-"${SPARK_VERSION}"-bin-hadoop3.tgz
+
+# Add localhost to known hosts for starting standalone spark cluster.
+ssh-keygen -t rsa -f "$HOME/.ssh/id_rsa" -P ""
+cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
```

Comment:
Is it possible to check whether this operation has been done? Or after running this script, developers may find that they cannot ssh onto other previously accessible remote machines, given that their public key has changed.